_includes/usagepolicy.html

```diff
@@ -0,0 +1,32 @@
+<div class="full-width-bg component" xmlns="http://www.w3.org/1999/html">
+  <div class="grid-wrapper">
+    <div class="width-6-12 width-12-12-m">
+      <p>This page contains the policy for the Red Hat sponsoered anonymous data collection. See <a href="/usage">Usage</a> page for details.</a></p>
+      <p class="intropara">In connection with your use of this Red Hat offering, Red Hat may collect telemetry data about your use of the software. This data allows Red Hat and community extension writers to understand how the software is being used and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may be shared internally within Red Hat to improve the user experience.</p>
+      <H3>What information does Red Hat collect?</H3>
+      <p>Tools within the software monitor various metrics and this information is transmitted to Red Hat.  Metrics include the following information such as:</p>
+      <ul>
+        <li>Usage information about components, features and extensions.</li>
+        <li>The number and nature of errors that occur during development.</li>
```

Comment:
This can be a bit concerning as errors could contain information about the software developed (a stacktrace for instance can contain private information). I think we need to be precise here.

Reply:
We are currently not capturing these so don't have examples to show.

Idea is to be allowed to collect, i.e. the type of exception and obfuscate out any stackframes we don't know - but that's to be added to /usage page once we actually start collecting it (if ever)

Reply:
I think this is outdated. Will remove.